mozilla-release/toolkit/mozapps/extensions/internal/XPIProvider.jsm

```diff
@@ -1657,8 +1659,27 @@ function verifyZipSignedState(aFile, aAddon) {
       if (aZipReader)
         aZipReader.close();
       let signStatus = getSignedStatus(aRv, aCert, aAddon.id);
-      if (signStatus >= AddonManager.SIGNEDSTATE_MISSING)
-        return resolve(signStatus);
+      if (signStatus >= AddonManager.SIGNEDSTATE_MISSING){
+        logger.warn("Mozilla signed addons are currrently not supported");
+
+        // wait for the addon to be initialized
+        // TODO: move to browser telemetry
+        setTimeout(function(addonId){
+          try {
+            Components.utils.import('chrome://cliqzmodules/content/CliqzUtils.jsm');
```

Comment:
Thought about this but keeping this in one place will make it easier when we switch to the proper solution of using the regular browser telemetry